demos/projects/ESPRESSIF/esp32/config/demo_config.h

```diff
@@ -0,0 +1,174 @@
+/* Copyright (c) Microsoft Corporation.
+   Licensed under the MIT License. */
+
+#ifndef DEMO_CONFIG_H
+#define DEMO_CONFIG_H
+
+/* FreeRTOS config include. */
+#include "FreeRTOSConfig.h"
+#include <stdlib.h>
+
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Include logging header files and define logging macros in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
+ * the logging configuration for DEMO.
+ * 3. Include the header file "logging_stack.h", if logging is enabled for DEMO.
+ */
+
+// TODO (ewertons): revisit the dependency on FreeRTOS-Plus, including this headers (logging)
+#include "logging_levels.h"
+
+#define STRING2(x) #x
+#define STRING(x) STRING2(x)
+
+/* Logging configuration for the Demo. */
+#ifndef LIBRARY_LOG_NAME
+    #define LIBRARY_LOG_NAME    "AzureIoTDemo"
+#endif
+
+#ifndef LIBRARY_LOG_LEVEL
+    #define LIBRARY_LOG_LEVEL    LOG_INFO
+#endif
+
+/* 
+ * The function prints to the console before the network is connected;
+ * then a UDP port after the network has connected. */
+extern void vLoggingPrintf( const char * pcFormatString,
+                            ... );
+
+/* Map the SdkLog macro to the logging function to enable logging */
+#ifndef SdkLog
+    #define SdkLog( message )    vLoggingPrintf message
+#endif
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief Enable Device Provisioning
+ * 
+ * @note To disable Device Provisioning undef this macro
+ *
+ */
+
+#ifdef CONFIG_ENABLE_DPS_SAMPLE
+#define democonfigENABLE_DPS_SAMPLE
```

Comment:
nit: 4 spaces

Reply:
This is about indentation... [to be done].